cmd/spire/messages.go

```diff
@@ -248,13 +248,10 @@ func handleMuSigMessage(msg *messages.MuSigMessage) streamType {
 			})
 		}
 
-		for _, optimistic := range msgTickMeta.Optimistic {
-			signatures = append(signatures, map[string]any{
-				"type":         "scribe-optimistic/v1",
-				"signature":    optimistic.ECDSASignature.String(),
-				"signers_blob": hexutil.BytesToHex(optimistic.SignerIndexes),
-			})
-		}
+		signatures = append(signatures, map[string]any{
+			"type":      "scribe-optimistic/v1",
```

Comment:
```suggestion
			"type":      "scribe-optimistic/v2",
```
I think, since we changed scribe to v2, we might also bump this.